aten/src/ATen/native/quantized/cpu/qmul.cpp

```diff
@@ -0,0 +1,91 @@
+#include <ATen/ATen.h>
+#include <ATen/core/op_registration/op_registration.h>
+#include <ATen/native/TensorIterator.h>
+#include <ATen/native/cpu/Loops.h>
+#include <ATen/quantized/Quantizer.h>
+
+#include <algorithm>
+
+namespace at {
+namespace native {
+namespace {
+
+inline void check_inputs(const Tensor& qa, const Tensor& qb) {
+  TORCH_CHECK(qa.qscheme() == kPerTensorAffine ||
+              qa.qscheme() == kPerTensorSymmetric,
+              "Only per tensor quantization is suuported in Mul.");
+  TORCH_CHECK(qa.qscheme() == qb.qscheme(),
+              "Both inputs to Mul must have the same quantization shceme.");
+  TORCH_CHECK(qa.numel() == qb.numel(),
```

Comment:
Why do we not support broadcast? For float, this works:
```
z = torch.rand(3,4,5)
v = torch.rand(3,1,1)
q = torch.mul(z,v)
``` 
and we should aim to match the same.

Reply:
This works because there is a broadcasting logic there. It is in the todo list to implement the broadcasting for the quantized ops, but not part of this PR.